index.html

```diff
@@ -28,10 +28,159 @@ <h1>Muhammad Nisar Ali</h1>
                     or website. Take a look of my works. If you like what you see and have a
                     project you need coded, don't hesitate and contact me.
                 </p>
+                <button type="button" class="button collab-button">Start collaboration</button>
             </div>
-            <button class="button collab-button">Start collaboration</button>
         </section>
-        <div class="bottom-img"></div>
+        <div class="bottom-img">
+            <img class="bg-bottom" src="./Images/image_geometry_2.svg" alt="Background Image">
+        </div>
+
+        <section class="proj-section">
+            <h2 class="heading">Projects</h2>
+
+            <article class="proj-container">
+                <img class="proj-img" src="./Images/Rectangle 21 (1).svg" alt="">
+
+                <div class="proj-description">
+                    <h3 class="proj-name">Project name goes here</h3>
+
+                    <ul class="languages">
+                        <li class="lang-used">HTML/CSS</li>
+                        <li class="lang-used">Ruby on Rails</li>
+                        <li class="lang-used">JavaScript</li>
+                    </ul>
+
+                    <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>
+                </div>
+            </article>
+            
+            <article class="proj-container">
+                <img class="proj-img" src="./Images/Rectangle 21.svg" alt="">
+
+                <div class="proj-description">
+                    <h3 class="proj-name">Project name goes here</h3>
+
+                    <ul class="languages">
+                        <li class="lang-used">HTML/CSS</li>
+                        <li class="lang-used">Ruby on Rails</li>
+                        <li class="lang-used">JavaScript</li>
+                    </ul>
+
+                    <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>
+                </div>
+            </article>
+
+            <article class="proj-container">
+                <img class="proj-img" src="./Images/Rectangle 21 (2).svg" alt="">
+
+                <div class="proj-description">
+                    <h3 class="proj-name">Project name goes here</h3>
+
+                    <ul class="languages">
+                        <li class="lang-used">HTML/CSS</li>
+                        <li class="lang-used">Ruby on Rails</li>
+                        <li class="lang-used">JavaScript</li>
+                    </ul>
+
+                    <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>
```

Comment:
Please iimplement the hover and click functionality for each button ` <button class="proj-link" type="button"> See this project <img src="./Images/ic_arrow_right.svg" alt="Click and proced to check the project"></button>` in your project just like you have done for the **"start collaboration "** buttons

Reply:
Thanks for your review. I have made changes as per project requirements.